content/Tjenester/samtykke.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Fiks samtykke
+date: 2024-09-12
+alias: [/fiks-plattform/tjenester/samtykke/]
+---
+
+## Kort beskrivelse
+
+Fiks samtykke er en løsning laget for Norges innbyggere for å kunne ta stilling til ulike samtykker. Løsningen består av en web-portal brukt av innbyggere til å ta stilling til samtykke og et API for eksterne fag-system brukt saksbehandlere til å opprette og endre samtykker.
+
+Første iterasjon av denne tjenesten omfatter samtykker knyttet til Barnevern, og det er den løsningen som i hovedsak beskrives her.
+
+## Tilgjengelige grensesnitt
+| Grensesnitt | Støtte |
+|------|------|
+| Web portal | Ja |
+| Maskin til maskin | Ja |
+
+## Beskrivelse av api
+### Statuser
+Det er to typer statuser som er definert i samtykke-løsningen. En overordnet samtykkeStatus per samtykke og en samtykkeSvarStatus per part.
+
+#### SamtykkeStatus
+SamtykkeStatus kan være følgende verdier:
+- SAMTYKKET("Samtykket"): Forteller at alle partene for et samtykke har samtykket.
```

Comment:
Kan vi kun bruke enum-navnet her?
SAMTYKKET: Forteller at alle pratene for et samtykke har samtykket.